### ISSUE TITLE
Lidarr not importing files after successful download

### ISSUE THREAD

**Issue by sauravkghosh** [lidarr/Lidarr#860]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Logs**
Link to debug logs.

**System info (please complete the following information):**
 - Lidarr Version: [e.g. 0.3.0.430]
 - Operating System [e.g. iOS]
 - .net Framework (Windows) or mono (macOS/Linux) Version: [e.g. 4.5 or 5.12]

**Additional context**

**Comment by GrayHogan** [comment]
This issue isn't really filled out.

**Comment by sauravkghosh** [comment]
After a successful download on a remote torrent client, Lidarr grabs it and puts it into the /downloads/ folder (this is a docker image of lidarr).  Even though it knew the 'output folder' is "/downloads/Iggy Azalea - In My Defense (2019) Mp3 (320 kbps) [Hunter]" it complains in the logs that "19-7-18 21:14:47.5|Error|DownloadedTracksImportService|Import failed, path does not exist or is not accessible by Lidarr: /downloads/Iggy Azalea - In My Defense"

In other words, it's not looking for the right folder, even though it was expecting the correct one.  Log is attached.

[lidarr.txt](https://github.com/lidarr/Lidarr/files/3411875/lidarr.txt)

**Comment by sauravkghosh** [comment]
Version: 0.6.2.883
Mono Version: 5.18.1.0
Docker: True
DB Migration: 29

**Comment by sauravkghosh** [comment]
> This issue isn't really filled out.

Wow -- you're really fast!! I was collecting the info and writing up the issue.

**Comment by GrayHogan** [comment]
Sorry, saw it in the discord channel heh

**Comment by ta264** [comment]
This is just a config error. Use the discord channel for support please.
